package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "nimiq-popup-shop",
+  "version": "1.0.0",
+  "description": "Client side, decentralized, minimal shop solution using IPFS and the Nimiq blockchain.",
+  "main": "nimiq-shop.js",
+  "module": "nimiq-shop.js",
+  "scripts": {
+    "start": "webpack-dev-server --config ./webpack.development.js",
+    "build": "webpack --config ./webpack.production.js",
+    "test": "karma start --single-run --browsers ChromeHeadless karma.conf.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/svub/nimiq-popup-shop.git"
+  },
+  "keywords": [
+    "nimiq",
+    "ipfs",
+    "shop",
+    "javascript",
+    "client-side"
+  ],
+  "author": "Sven Buschbeck",
```

```suggestion
  "author": "Sven Buschbeck, Jens ?",
```
Put your name :) - if you don't mind.